common/proto-conv/tests/it/proto_conv.rs

```diff
@@ -236,37 +237,48 @@ fn test_load_old() -> anyhow::Result<()> {
-            51, 26, 15, 10, 10, 10, 5, 26, 0, 160, 6, 1, 160, 6, 1, 160, 6, 1, 160, 6, 1, 10, 16,
-            10, 4, 98, 111, 111, 108, 26, 5, 18, 0, 160, 6, 1, 160, 6, 1, 10, 16, 10, 4, 105, 110,
-            116, 56, 26, 5, 26, 0, 160, 6, 1, 160, 6, 1, 10, 17, 10, 5, 105, 110, 116, 49, 54, 26,
-            5, 34, 0, 160, 6, 1, 160, 6, 1, 10, 17, 10, 5, 105, 110, 116, 51, 50, 26, 5, 42, 0,
-            160, 6, 1, 160, 6, 1, 10, 17, 10, 5, 105, 110, 116, 54, 52, 26, 5, 50, 0, 160, 6, 1,
-            160, 6, 1, 10, 17, 10, 5, 117, 105, 110, 116, 56, 26, 5, 58, 0, 160, 6, 1, 160, 6, 1,
-            10, 18, 10, 6, 117, 105, 110, 116, 49, 54, 26, 5, 66, 0, 160, 6, 1, 160, 6, 1, 10, 18,
-            10, 6, 117, 105, 110, 116, 51, 50, 26, 5, 74, 0, 160, 6, 1, 160, 6, 1, 10, 18, 10, 6,
-            117, 105, 110, 116, 54, 52, 26, 5, 82, 0, 160, 6, 1, 160, 6, 1, 10, 19, 10, 7, 102,
-            108, 111, 97, 116, 51, 50, 26, 5, 90, 0, 160, 6, 1, 160, 6, 1, 10, 19, 10, 7, 102, 108,
-            111, 97, 116, 54, 52, 26, 5, 98, 0, 160, 6, 1, 160, 6, 1, 10, 16, 10, 4, 100, 97, 116,
-            101, 26, 5, 106, 0, 160, 6, 1, 160, 6, 1, 10, 26, 10, 9, 116, 105, 109, 101, 115, 116,
-            97, 109, 112, 26, 10, 114, 5, 8, 5, 160, 6, 1, 160, 6, 1, 160, 6, 1, 10, 18, 10, 6,
-            115, 116, 114, 105, 110, 103, 26, 5, 122, 0, 160, 6, 1, 160, 6, 1, 10, 46, 10, 6, 115,
-            116, 114, 117, 99, 116, 26, 33, 130, 1, 27, 10, 3, 102, 111, 111, 10, 3, 98, 97, 114,
-            18, 5, 18, 0, 160, 6, 1, 18, 5, 122, 0, 160, 6, 1, 160, 6, 1, 160, 6, 1, 160, 6, 1, 10,
-            28, 10, 5, 97, 114, 114, 97, 121, 26, 16, 138, 1, 10, 10, 5, 18, 0, 160, 6, 1, 160, 6,
-            1, 160, 6, 1, 160, 6, 1, 10, 23, 10, 7, 118, 97, 114, 105, 97, 110, 116, 26, 9, 146, 1,
-            3, 160, 6, 1, 160, 6, 1, 160, 6, 1, 10, 29, 10, 13, 118, 97, 114, 105, 97, 110, 116,
-            95, 97, 114, 114, 97, 121, 26, 9, 154, 1, 3, 160, 6, 1, 160, 6, 1, 160, 6, 1, 10, 30,
-            10, 14, 118, 97, 114, 105, 97, 110, 116, 95, 111, 98, 106, 101, 99, 116, 26, 9, 162, 1,
-            3, 160, 6, 1, 160, 6, 1, 160, 6, 1, 10, 26, 10, 8, 105, 110, 116, 101, 114, 118, 97,
-            108, 26, 11, 170, 1, 5, 8, 2, 160, 6, 1, 160, 6, 1, 160, 6, 1, 18, 6, 10, 1, 97, 18, 1,
-            98, 160, 6, 1, 42, 10, 10, 3, 120, 121, 122, 18, 3, 102, 111, 111, 50, 2, 52, 52, 58,
-            10, 10, 3, 97, 98, 99, 18, 3, 100, 101, 102, 74, 10, 40, 97, 32, 43, 32, 50, 44, 32,
-            98, 41, 162, 1, 23, 50, 48, 49, 52, 45, 49, 49, 45, 50, 56, 32, 49, 50, 58, 48, 48, 58,
-            48, 57, 32, 85, 84, 67, 170, 1, 23, 50, 48, 49, 52, 45, 49, 49, 45, 50, 57, 32, 49, 50,
-            58, 48, 48, 58, 49, 48, 32, 85, 84, 67, 178, 1, 13, 116, 97, 98, 108, 101, 95, 99, 111,
-            109, 109, 101, 110, 116, 160, 6, 1, 160, 6, 1,
+            10, 10, 8, 5, 16, 6, 160, 6, 1, 168, 6, 1, 18, 3, 102, 111, 111, 26, 3, 98, 97, 114,
```

Comment:
These data is for compatibility test that new program can load old data. Do not modify it.

Reply:
Fine